cmd/token.go

```diff
@@ -50,17 +50,32 @@ var (
 	}
 
 	tokenCreateCmd = &cobra.Command{
-		Use:   "create",
+		Use:   "create --read|-r | --write|-w | --modify|-m | --stage|-s <pelican-url>",
```

Comment:
Can you remove the flags enumeration? Invoking the CLI with `-h` already indicates the flags. It's only the required pelican URL that wasn't stated.